src/review/views.py

```diff
@@ -799,6 +825,66 @@ def do_review(request, assignment_id):
     return render(request, template, context)
 
 
+@require_POST
+@reviewer_user_for_assignment_required
+def upload_review_file(request, assignment_id):
+    access_code = logic.get_access_code(request)
```

Comment:
It'd make sense for the `reviewer_user_for_assignment_required` to pass the `access_code` as an argument instead of having to do this again here, but this is not an problem with this PR so I'll shut up :zipper_mouth_face:

Reply:
SHUSH but yes.